src/utils/metrics/prometheus.go

```diff
@@ -233,8 +231,26 @@ func getBasicAuth() (string, string, error) {
 // PushGatewayCA variable to embed self-signed CA for TLS
 var PushGatewayCA string
 
-// getTLSConfig returns tls.Config with system root CAs and embedded CA if not empty
-func getTLSConfig() (*tls.Config, error) {
+// getTLSConfig returns cached tls.Config with system root CAs and embedded CA if not empty
+func getTLSConfig() (config *tls.Config, err error) {
```

Comment:
If my memory serves me well, this is called only once by `go pushMetrics` or something like that at the start of the program (if it's not than it's a bug)
The key thing is that those metrics were introduced initially to have rough understanding of how many people are using this app to be able to plan attacks better but right now there are so many users that in order to keep up with that I'd need to scale up something like a full Thanos cluster so I'm thinking about removing push metrics completely

Reply:
> (if it's not than it's a bug)

Yes, it was called multiple times

Reply:
well, not an issue now anyway. the server is dead for couple weeks already so I just removed this code completely :)